draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
> This situation should only occur as a race condition between the answer and ClientHello, so any additional delay would be minimal.

I don't think it's safe to assume that the delay between the answer and ClientHello is minimal. They may go over different network paths, and there may be factors unrelated to DTLS that delay the generation of the answer.

And even if the delay between the answer and ClientHello is small, some applications may care about any small time savings. This isn't just a hypothetical issue; I've experimented with a non-standard ICE extensions that make this possible, for exactly this reason.

I accept that this is only legitimately possible with ICE-lite right now. But wouldn't it be feasible for a future RFC to introduce a new way in which it's possible? 

I guess I don't see the advantage of this restriction, besides making the spec simpler. Or is that the main advantage?